components/list/src/ListItem.js

```diff
@@ -0,0 +1,157 @@
+/* @jsx jsx */
+
+import React, { Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { jsx, css, useTheme } from '@westpac/core';
+import { List } from './List';
+
+// ==============================
+// Utils
+// ==============================
+const getListIcon = ({ appearance, color, icon: Icon }) => {
+	const { list } = useTheme();
+
+	const style = {
+		position: 'absolute',
+		left: 0,
+		top: 2,
+	};
+
+	switch (appearance) {
+		case 'bullet':
+			return (
+				<svg
+					aria-labelledby="title-bullet-list"
+					version="1.1"
+					xmlns="http://www.w3.org/2000/svg"
+					width="16px"
+					height="16px"
+					viewBox="0 0 16 16"
+					css={style}
+				>
+					<circle
+						r="4"
+						cx="8"
+						cy="8"
+						stroke={`${list[appearance][color].color}`}
+						strokeWidth="1"
+						fill={`${list[appearance][color].color}`}
+					/>
+				</svg>
```

Comment:
Do we want to consider the background image data-uri method for these? See the select element caret icons in `FormInput` component.

Keen to understand the pros/cons of each approach. There may be legacy support rationale to using background image rather than embedded svg.

Reply:
Also.... I see you've implemented via SVG rather than css shapes. What are the pros/cons?